<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-BIJL3HUR" ignored="0" name="loginTest" tests="1" failures="1" timestamp="2022-07-06T20:34:13 CDT" time="0.051" errors="0">
  <testcase name="validUserShouldBeAbleToAddCustomer" time="0.013" classname="test.AddContactTest">
    <failure type="org.testng.internal.reflect.MethodMatcherException" message="
[public void test.AddContactTest.validUserShouldBeAbleToAddCustomer(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String) throws java.lang.InterruptedException] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: validUserShouldBeAbleToAddCustomer([Parameter{index=0, type=java.lang.String, declaredAnnotations=[@org.testng.annotations.Optional(value=default parameter will execute if the parameter tag misses in the testng.xml )]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=2, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=3, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=4, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=5, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=6, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=7, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=8, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=9, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=10, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=11, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) demo@Techfios.com,(java.lang.String) abc123,(java.lang.String) Excel Projet,(java.lang.String) Techfios,(java.lang.String) msexcell@gmail.com,(java.lang.String) 231546,(java.lang.String) Techfios Building,(java.lang.String) Carrolton,(java.lang.String) United Kingdom,(java.lang.String) TX,(java.lang.String) 75019]">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void test.AddContactTest.validUserShouldBeAbleToAddCustomer(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String) throws java.lang.InterruptedException] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Arguments: [(java.lang.String) demo@Techfios.com,(java.lang.String) abc123,(java.lang.String) Excel Projet,(java.lang.String) Techfios,(java.lang.String) msexcell@gmail.com,(java.lang.String) 231546,(java.lang.String) Techfios Building,(java.lang.String) Carrolton,(java.lang.String) United Kingdom,(java.lang.String) TX,(java.lang.String) 75019]
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- validUserShouldBeAbleToAddCustomer -->
</testsuite> <!-- loginTest -->
